### PR TITLE
Removes use of unwanted Team table

### PR DESCRIPTION
**Pull request by sudheesh001**
Work done here
- Removes the Team table because its not being used anywhere.
- Deleted the original SQLite DB with the Teams table and recreated it, fixed errors in env and server while doing that.
- Removes whitelist authentication failures because of hanging env params
- Kept the database to be standard as `sprinter.sqlite` instead of making it variable

Signed off by: Sudheesh Singanamalla <ssingana@redhat.com>

### CODE REVIEW

File: server/server.js [hunk 1]
```diff
@@ -1,16 +1,7 @@
 module.exports = function (env) {
```
Comment (debloper) [inline server/server.js]: The `env` argument can/should be removed now, as decommissioned.

File: server/app.js [hunk 2]
```diff
@@ -6,7 +6,7 @@ Habitat.load();
 var env = new Habitat();
 
 // App
-var app = require('./server')(env);
+var app = require('./server')('sprinter.sqlite');
```
Comment (debloper) [inline server/app.js]: No need of the argument, as it's already hard-coded in `server.js`
Please verify the changes minutely (setup a fresh sandbox staging, if required) to see if everything works.

**Comment by sudheesh001** [comment]
@debloper Done. Fixed them. Works perfectly.

**Comment by debloper** [comment]
Thanks, patch merged!

**Pull request merged**